src/chrome/content/scaffold/scaffold.js

```diff
@@ -108,6 +111,12 @@ var Scaffold = new function() {
 		_editors["tests"].getSession().setUseSoftTabs(false);
 		
 		_editors["import"].getSession().setMode(new importWin.TextMode);
+		
+		// Set font size from previous session
+		var size = Zotero.Prefs.get("scaffold.fontSize");
```

Comment:
@dstillman Please check whether it is okay to use Zotero preferences here.

Reply:
Ping @dstillman

Reply:
Okay, good idea with this function. Is the argument then also the same, i.e. `Zotero.Prefs.clear("scaffold.fontSize")`?

Reply:
Yep.